### PR TITLE
update concurrency support

### PR DESCRIPTION
**Pull request by valentyn88**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Description

We support concurrency for Cypress, TestCafe, and Playwright. The default value for concurrency from now is equal to 2. 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by spider-sauce — APPROVED** [review]
LG2M 👍

File: docs/testrunner-toolkit/configuration.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ https://github.com/saucelabs/testrunner-toolkit/blob/master/.sauce/testcafe.yml
 </Tabs>
 
 ## Concurrency
-<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small> , <small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight></small></p>
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small> , <small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight> <Highlight color="#25c2a0">testcafe</Highlight> <Highlight color="#25c2a0">playwright</Highlight></small></p>
```
Comment (alexplischke) [inline docs/testrunner-toolkit/configuration.md:127]: This is not true anymore, we support concurrency in both docker and sauce cloud ⚠️

File: docs/testrunner-toolkit/configuration/common-syntax.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ __Example__:
 ```
 
 ### `concurrency`
-<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small> , <small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight></small></p>
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small> , <small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight> <Highlight color="#25c2a0">testcafe</Highlight> <Highlight color="#25c2a0">playwright</Highlight></small></p>
```
Comment (alexplischke) [inline docs/testrunner-toolkit/configuration/common-syntax.md:82]: Same here ⚠️

File: docs/testrunner-toolkit/configuration.md (hunk 1 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration.md:127]: I created a new PR to update it.

File: docs/testrunner-toolkit/configuration/common-syntax.md (hunk 2 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration/common-syntax.md:82]: I created a new PR to update it.

**Pull request merged**